src/doc/src/reference/workspaces.md

```diff
@@ -222,6 +223,33 @@ cc.workspace = true
 rand.workspace = true
 ```
 
+## The `lints` table
+
+The `workspace.lints` table is where you define lint configuration to be inherited by members of a workspace.
+
+Specifying a workspace lint configuration is similar to package lints.
+
+Example:
+
+```toml
+# [PROJECT_DIR]/Cargo.toml
+[workspace]
+members = ["crates/*"]
+
+[workspace.lints.rust]
+unsafe_code = "forbid"
```

Comment:
It seems that both `unsafe_code` and `unsafe-code` works. It may not be worth a mention in doc but the error message always shows dash version, which is a bit unfortunate. 

```
  = note: requested on the command line with `-D unsafe-code`
```